ovos_plugin_manager/templates/triples.py

```
@@ -1,14 +1,346 @@
+            subject: str,
+            predicate: str,
+            obj: str,
+            confidence: float = 1.0,
+            metadata: Optional[Dict[str, Any]] = None) -> Triple:
+        """Persist a single triple.
+
+        Args:
+            subject: Triple subject.
+            predicate: Triple predicate.
+            obj: Triple object.
+            confidence: Confidence score (0–1).
```

Ruff (0.15.7)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)